.github/workflows/main.yml

```diff
@@ -189,6 +189,79 @@ jobs:
         TEST_CASE: ${{ matrix.TEST_CASE }}
         TARGET: ${{ matrix.TARGET }}
 
+  opentitan-earl-grey-sim:
+    runs-on: ubuntu-latest
+    needs: build-binaries
+    env:
+      CC: gcc-9
+      CXX: g++-9
+
+    steps:
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update -qq
+        sudo apt install -y g++-9 build-essential cmake tclsh ant default-jre swig google-perftools libgoogle-perftools-dev python3 python3-dev uuid uuid-dev tcl-dev flex libfl-dev libelf-dev ninja-build
+
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 1
+
+    - uses: actions/setup-python@v2
+      with:
+         python-version: '3.7'
+
+    - name: Download binaries
+      uses: actions/download-artifact@v2
+      with:
+        name: binaries
+
+    # See https://github.com/actions/upload-artifact/issues/38
+    - name: Extract
+      run: tar -xf binaries.tar
+
+    - name: Prepare output directory
+      run: mkdir output_uhdm
```

Comment:
Please rename output directory, as it contains also vanilla verilator output.

Reply:
Renamed to `output_uhdm_vanilla_verilator`